### ISSUE TITLE
Regel: Klargjøre antall spillere på benken (4.1.1) 

### ISSUE THREAD

**Issue by hansfn** [nvbf/spilleregler#11]
[4.1.1](https://spilleregler.volleyball.no/para4/#r4-1-1) sier:

> Et lag kan maksimalt bestå av maksimalt 12 spillere [...]

men i fotnoten så står det:

> I verdensomfattende FIVB og offisielle konkurranser er det tillatt med maksimalt 14 spillere

I de engelske reglene står det:

> For FIVB, World and **Official competitions for Seniors**:
> Up to 14 players may be recorded on the score sheet and play in a match.

Jeg mener at alle våre seriekamper er "offisiellle konkurranser for senior" og dermed er det bedre hvis 14 spillere er standard og ikke skjult i en uklar fotnote. Når er det egentlig ikke lov med 14 spillere - US-runder? Under junior-NM i U17 og U19 tillates i alle fall 14 spillere.

**Comment by VelimirPetrovic** [comment]
For det min mening er verdt, er jeg helt enig med Hans Fredrik her. Jeg mener også at alle offisielle seniorkamper og alle junior-NM-kamper jeg har vært på, har vært slik Hans Fredrik beskriver. Dvs. 14 spillere totalt, 12+ spillere betyr 2 liberoer.

**Comment by vegardbus** [comment]
@hansfn - Er dette noko DK kan spele inn til reglementsutvalet f.eks? At ein generelt ynskjer 14 som gjeldande i Noreg.

**Comment by hansfn** [comment]
Ja, det kan jeg høre med DK om. Hvis DK er enig, så sender vi det videre.
